Site/ASAB Maestro/Files/influxdb/influx-init.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+# Wait for InfluxDB to start
+until curl -s http://{{INFLUXDB_HOSTNAME}}:8086/health | grep -q '"status": "pass"'; do
+  echo "Waiting for InfluxDB to start..."
+  sleep 1
+done
+
+# Get the bucket ID
+BUCKET_ID=$(curl -s -X GET http://{{INFLUXDB_HOSTNAME}}:8086/api/v2/buckets -H 'Authorization: Token "{{INFLUXDB_TOKEN}}"' -H "Accept: application/json" | jq -r '.buckets[] | select(.name=="{{DOCKER_INFLUXDB_INIT_BUCKET}}") | .id')
```

Comment:
Even though we create just one bucket, there can be multiple in the influx in the end if created manually by the user.